src/utils/result-list.js

```diff
@@ -3,19 +3,32 @@
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
 import { isFirefox } from "devtools-config";
+import {
```

Comment:
Weird that CI is tripping here. Maybe just leave the original name?

Reply:
done did it :)